Assets/MRTK/Providers/LeapMotion/package.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "com.microsoft.mixedreality.toolkit.leapmotion",
-    "displayName": "Mixed Reality Toolkit Leap Motion Provider",
-    "version": "2.5.0-preview.0",
+    "name": "com.microsoft.mixedreality.toolkit.leapmotion.legacy",
+    "displayName": "Mixed Reality Toolkit Leap Motion (Legacy XR)",
+    "version": "%version%",
     "unity": "2018.4",
-    "description": "Platform support for the UltraLeap Leap Motion on the Microsoft Mixed Reality Toolkit",
+    "description": "Platform support for the UltraLeap Leap Motion on the Microsoft Mixed Reality Toolkit (not supported on Unity 2020 and newer)",

```

Comment:
>(not supported on Unity 2020 and newer)

Is this true? I didn't think Leap Motion had a specific XR pipeline dependency. @CDiaz-MS?

Reply:
There is no specific dependency on the XR pipeline in the Leap Data Provider.

Reply:
@CDiaz-MS, to be clear, the leap provider uses _no_ Unity XR APIs?

Reply:
removed .legacy and the erroneous 2020 note

Reply:
Yes, no Unity XR APIs are used.  Only refs to MRTK and the Leap Core Assets package.